client/lib/site/index.js

```diff
@@ -86,46 +87,7 @@ Site.prototype.set = function( attributes ) {
  * Update any computed attributes
  */
 Site.prototype.updateComputedAttributes = function() {
-	/**
```

Comment:
@ehg : Sorry, I hadn't seen that you'd dropped a comment, but I'm not sure we're on the same page. I'm specifically referring to these newly introduced lint warnings:

```
./node_modules/.bin/eslint --quiet --rule 'no-unused-vars: 2' client/lib/site/index.js 

/Users/andrew/Documents/Code/wp-calypso/client/lib/site/index.js
   9:2  error  'trim' is defined but never used     no-unused-vars
  15:2  error  'config' is defined but never used   no-unused-vars
  19:2  error  'isHttps' is defined but never used  no-unused-vars

✖ 3 problems (3 errors, 0 warnings)
```